### PR TITLE
Add support for HA configuration of none and standard auth services

### PR DESCRIPTION
**Pull request by lulf**
### Type of change

- Enhancement / new feature
- Documentation

### Description

This PR implements high availability configuration for the none authservice and the standard authservice (when backed by postgres). The PR contains systemtests to verify the additional replicas being created and that auth still works.

The documentation is updated with an additional procedure for configuring the standard authservice in HA mode.

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [x] Write tests and make sure they pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [x] Reference relevant issue(s) and close them after merging
- [x] Update CHANGELOG.md

### CODE REVIEW

**Comment by lulf** [comment]
@enmasse-ci run tests profile=isolated testcase=AuthServiceTest

**Comment by lulf** [comment]
@enmasse-ci run tests profile=isolated testcase=AuthServiceTest

**Comment by lulf** [comment]
@enmasse-ci run tests profile=isolated testcase=AuthServiceTest

**Comment by lulf** [comment]
@enmasse-ci run tests profile=isolated testcase=AuthServiceTest

**Comment by lulf** [comment]
@enmasse-ci run tests profile=isolated testcase=AuthServiceTest

**Comment by lulf** [comment]
@enmasse-ci run tests profile=isolated testcase=AuthServiceTest

**Comment by lulf** [comment]
@enmasse-ci run tests profile=isolated

File: documentation/modules/proc-standard-auth-service-ha.adoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// assembly-deploying-auth-services.adoc
+
+[id='proc-deploying-standard-auth-service-{context}']
+= Deploying the `standard` authentication service for HA
+
+To implement the `standard` authentication service, you deploy it.
```
Comment (hmaclean73) [inline documentation/modules/proc-standard-auth-service-ha.adoc:8]: Perhaps include a brief sentence describing why the user should perform this procedure?

File: documentation/modules/proc-standard-auth-service-ha.adoc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+.Prerequisites
+* A PostgreSQL Database.
```
Comment (hmaclean73) [inline documentation/modules/proc-standard-auth-service-ha.adoc:11]:
```suggestion
* A PostgreSQL database.
```

File: documentation/modules/proc-standard-auth-service-ha.adoc [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+ifeval::["{cmdcli}" == "oc"]
+. Log in as a service admin:
++
+[subs="attributes",options="nowrap"]
+----
+{cmdcli} login -u admin
+----
+endif::[]
+. Create a `Secret` with the database credentials:
```
Comment (hmaclean73) [inline documentation/modules/proc-standard-auth-service-ha.adoc:23]:
```suggestion
. Create a secret with the database credentials:
```

**Comment by lulf** [comment]
@enmasse-ci run tests profile=isolated

File: documentation/modules/proc-standard-auth-service-ha.adoc [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+[subs="attributes",options="nowrap"]
+----
+{cmdcli} login -u admin
+----
+endif::[]
+. Create a secret with the database credentials:
++
+[source,yaml,options="nowrap"]
+----
+{cmdcli} create secret generic db-creds -n _{ProductNamespace} --from-literal=database-user=_admin_ --from-literal=database-user=_securepassword_
```
Comment (robshelly) [inline documentation/modules/proc-standard-auth-service-ha.adoc:27]: Both keys are database-user

**Pull request merged**